types/product.js

```diff
@@ -156,10 +156,10 @@ export type Variant = {
   id: number,
   sku?: string,
   key?: string,
-  prices: Array<Price>,
-  images: Array<Image>,
-  attributes: Array<?Attribute>,
-  assets: Array<?Asset>,
+  prices?: Array<Price>,
+  images?: Array<Image>,
+  attributes?: Array<?Attribute>,
+  assets?: Array<?Asset>,
```

Comment:
these are all optional as of the [api spec](https://docs.commercetools.com/http-api-projects-products#productvariant).